test-app/ember-cli-build.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+const EmberApp = require('ember-cli/lib/broccoli/ember-app');
```

Comment:
Not sure how to fix this, as this dep is defined in the [test app's package.json](https://github.com/ember-mocks/ember-date-service/pull/174/files#diff-b03f4acb30e04ceb12a40763ef649759b3f632df2357bcec14fab6dd29ebbfb4R28), but doesn't get installed in the `test-app/node_modules`